uaclient/status.py

```diff
@@ -245,6 +245,9 @@ class UserFacingStatus(enum.Enum):
 MESSAGE_NO_ACTIVE_OPERATIONS = """No Ubuntu Advantage operations are running"""
 MESSAGE_LOCK_HELD = """Operation in progress: {lock_holder} (pid:{pid})"""
 PROMPT_YES_NO = """Are you sure? (y/N) """
+MESSAGE_REBOOT_SCRIPT_FAILED = (
+    "Failed running reboot_cmds script.\n" "See: /var/log/ubuntu-advantage.log"
```

Comment:
Why the separation of sub-strings?

Reply:
My mistake here, I thought the string was going to be grater than 80 characters and it is not. I will update the code